dvc/command/experiments.py

```diff
@@ -783,6 +784,13 @@ def add_parser(subparsers, parent_parser):
         help="Exclude the specified params from output table.",
         metavar="<params_list>",
     )
+    experiments_show_parser.add_argument(
+        "-t",
+        "--tracked-params",
```

Comment:
They're all tracked (some by Git). What about `--deps` or `--only-deps` (or `--param-deps` in this case)

Context: https://github.com/iterative/dvc/issues/5451#issuecomment-794894902